### ISSUE TITLE
Better Brid.gy integration

### ISSUE THREAD

**Issue by pfefferle** [pfefferle/wordpress-webmention#329]
@snarfed do you think there is a way to better integrate the brid.gy settings page into the Webmention settings area? One understanding of the [Cloudfest Hackathon](https://www.cloudfest.com/project/wordpress-and-the-indieweb) was, that Brid.gy is an awesome extension and should be better featured.

/cc @moritzbappert @jaz-on @hagengraf

**Comment by snarfed** [comment]
Thanks for the kind words!

@dshanske built the [now-deprecated Bridgy WordPress plugin](https://wordpress.org/plugins/bridgy-publish/) ([more](https://indieweb.org/WordPress_with_Bridgy)), his experience is probably the most useful here. Otherwise, you could maybe use [Bridgy's API](https://brid.gy/about#api) to expose current account(s) status, last webmention timestamps, etc?

**Comment by dshanske** [comment]
I think once we finish the other enhancements, we can add some of the features of the api. I rolled the Bridgy Publish functionality into Syndication Links.

**Comment by jaz-on** [comment]
Indeed, the vision during the hackathon was to have a "click and go" plugin that allow a user to implement webmentions into their WordPress website in a minute. Presently, the webmention part is implemented by Matthias' plugin.

But it could be great if users could go further by setting up a bridge from the WordPress back-office (via the webmention plugin settings page).
I'm not saying it will be easy or how we can do that. I'm just saying that if it's possible to activate a bridge directly when installing the plugin on WordPress, it would make things much simpler.

Does it make sense what I'm saying? :)

**Comment by dshanske** [comment]
@jaz-on Automatically may be hard... but with entering in some information... definitely

**Comment by pfefferle** [comment]
Maybe we can at least start with a small how to in the settings and improve from there...

**Comment by snarfed** [comment]
@jaz-on yes! Afaik @dshanske's plugin did exactly that: it let you click a button in your site's WP admin that automatically set up your Bridgy account and then returned to the WP admin screen. Bridgy has a specific API for this: https://brid.gy/about#registration-api , which other services like [Known](https://withknown.com/) used too.

**Comment by pfefferle** [comment]
Is there an API to get the list of all possible services, so we do not have to hardcore them?

**Comment by dshanske** [comment]
I think a list of services would be helpful if it could be returned or parsed out

**Comment by snarfed** [comment]
I'd just hard code the list of services. It's small, just 7 right now, and it changes rarely, maybe once a year on average.

**Comment by snarfed** [comment]
If you really want to scrape them from the https://brid.gy/ front page, you're welcome to. No guarantees, it's not part of [the official API](https://brid.gy/about#api), but its HTML structure pretty much never changes.
